alertmanager/config/alertmanager.yml.tmpl

```diff
@@ -20,7 +20,7 @@ receivers:
     send_resolved: true
     title: '{{ template "slack.monzo.title" . }}'
     pretext: '{{ template "slack.monzo.pretext" . }}'
-    text: '{{ template "slack.monzo.text" . }}'
+    text: '<!channel> {{ template "slack.monzo.text" . }}'
```

Comment:
Shall we add this to the slack template file instead?

Reply:
done...